saleor/discount/models.py

```diff
@@ -171,6 +190,13 @@ def validate_only_for_staff(self, customer: Optional["User"]):
             raise NotApplicable(msg)
 
 
+class VoucherCode(models.Model):
```

Comment:
I am not sure, but I recall some discussion that all new models should have UUID